presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -1433,6 +1434,38 @@ public void testMaterializedViewOptimizationWithDerivedFieldsWithAlias()
         }
     }
 
+    @Test(enabled = false)
```

Comment:
Lets add a comment on what needs to be done to enable this test

Reply:
Does this test fail currently? What is the error message?

Reply:
@yuanzhanhku It does not fail. Since the base to views mapping cannot be fetched from the metastore, it will not rewrite the base query and return the original one. Therefore, the computeActual functions on lines 1461-1462 will be evaluating the same query in an identical way.

Currently we haven't figured out a better way to test if a base query is successfully rewritten or not, so the result comparison is merely a placeholder for the test case.

Reply:
@jzzhaofb The correct way to verify if the query has been rewritten or not is by doing the plan comparison. Lets try to debug on Monday the issues you are facing in implementing the plan assertion.